csrank/objectranking/cmp_net.py

```diff
@@ -39,7 +39,53 @@ def __init__(self, n_features, n_hidden=2, n_units=8,
                  kernel_regularizer=l2(l=0.02), non_linearities='relu',
                  optimizer=Adam(), metrics=[top_k_categorical_accuracy, binary_accuracy],
                  use_early_stopping=False, es_patience=300, batch_size=256, random_state=None, **kwargs):
-
+        """
+            Create an instance of the CmpNet architecture.
+
+            CmpNet breaks the rankings into pairwise comparisons and learns a
+            pairwise model for the each pair of object in the ranking.
```

Comment:
We should add here, that we aggregate in a different way than SortNet (sum)

Reply:
For predictions ?

Reply:
Changed it